### PR TITLE
Adds QCElemental

### PR DESCRIPTION
**Pull request by dgasmith**
Adds QCElemental, should be ready for review @conda-forge/help-python.

### CODE REVIEW

File: recipes/qcelemental/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    url: https://github.com/MolSSI/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    noarch: python
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    build:
```
Comment (beckermr) [inline recipes/qcelemental/meta.yaml]: This should be host

File: recipes/qcelemental/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    home: https://github.com/MolSSI/QCElemental
+    dev_url: https://github.com/MolSSI/QCElemental
+    doc_url: https://qcelemental.readthedocs.io/en/latest/
+    doc_source_url: https://github.com/MolSSI/QCElemental/blob/master/docs/source/index.rst
+    license: BSD-3-Clause
+    license_url: https://opensource.org/licenses/BSD-3-Clause
+    license_file: LICENSE
+    license_family: BSD
+    summary: QCElemental is a resource module for quantum chemistry containing physical constants and periodic table data from NIST and molecule handlers.
+    # description: >
```
Comment (beckermr) [inline recipes/qcelemental/meta.yaml]: Remove this comment.

File: recipes/qcelemental/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+extra:
+    recipe-maintainers:
+        - loriab
```
Comment (beckermr) [inline recipes/qcelemental/meta.yaml]: @loriab should respond here that they are OK being a maintainer.

**Comment by dgasmith** [comment]
@beckermr Thanks for the comments, updated.

File: recipes/qcelemental/meta.yaml (hunk 3 above)
Comment (loriab) [reply recipes/qcelemental/meta.yaml]: Affirmative.

**Comment by beckermr** [comment]
We cannot merge until the build passes.

**Comment by beckermr** [comment]
I added pip since it was needed. Sorry I missed that!

**Comment by dgasmith** [comment]
Ah, I missed as well, thanks for this.

Everything is now passing successfully!

**Pull request merged**